manifest.json

```
@@ -19,22 +19,14 @@
         "email": "tituspijean@outlook.com"
     },
     "requirements": {
-        "yunohost": ">= 4.2"
+        "yunohost": ">= 4.2.8"
```

```suggestion
        "yunohost": ">= 11"
```

To be changed to YunoHost's actual Bullseye version.